selfdrive/car/honda/interface.py

```diff
@@ -278,27 +277,27 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
       raise ValueError(f"unsupported car {candidate}")
 
     # These cars use alternate user brake msg (0x1BE)
-    if 0x1BE in fingerprint[CAN.pt] and candidate in HONDA_BOSCH:
+    if 0x1BE in fingerprint[CAN.pt] and candidate.config.flags & HondaFlags.BOSCH:
       ret.flags |= HondaFlags.BOSCH_ALT_BRAKE.value
       ret.safetyConfigs[0].safetyParam |= Panda.FLAG_HONDA_ALT_BRAKE
 
     # These cars use alternate SCM messages (SCM_FEEDBACK AND SCM_BUTTON)
-    if candidate in HONDA_NIDEC_ALT_SCM_MESSAGES:
+    if candidate.config.flags & HondaFlags.NIDEC_ALT_SCM_MESSAGES:
       ret.safetyConfigs[0].safetyParam |= Panda.FLAG_HONDA_NIDEC_ALT
 
-    if ret.openpilotLongitudinalControl and candidate in HONDA_BOSCH:
+    if ret.openpilotLongitudinalControl and candidate.config.flags & HondaFlags.BOSCH:
       ret.safetyConfigs[0].safetyParam |= Panda.FLAG_HONDA_BOSCH_LONG
 
-    if ret.enableGasInterceptor and candidate not in HONDA_BOSCH:
+    if ret.enableGasInterceptor and candidate.config.flags & HondaFlags.BOSCH:
       ret.safetyConfigs[0].safetyParam |= Panda.FLAG_HONDA_GAS_INTERCEPTOR
 
-    if candidate in HONDA_BOSCH_RADARLESS:
+    if candidate.config.flags & HondaFlags.BOSCH_RADARLESS:
       ret.safetyConfigs[0].safetyParam |= Panda.FLAG_HONDA_RADARLESS
 
     # min speed to enable ACC. if car can do stop and go, then set enabling speed
     # to a negative value, so it won't matter. Otherwise, add 0.5 mph margin to not
     # conflict with PCM acc
-    ret.autoResumeSng = candidate in (HONDA_BOSCH | {CAR.CIVIC}) or ret.enableGasInterceptor
+    ret.autoResumeSng = bool(candidate.config.flags & HondaFlags.AUTORESUME_SNG) or ret.enableGasInterceptor
```

Comment:
Perhaps for an exception of one we don't need a whole flag?

Reply:
Hmm, I guess but I think ideally carcontoller doesn't even know what a civic is

Reply:
like ideally the platform config tells carcontroller and carstate everything it needs to know to control the car